TPI/juegoDeLaVida/solucion.cpp

```diff
@@ -3,7 +3,16 @@
 
 /********************************** EJERCICIO esValido **********************************/
 bool esValido(toroide t) {
-    return true;
+    bool res = false;
+
+    if (filas(t) > 0 && columnas(t) > 0) {
+        res = true;
+        for (int f = 1; f < filas(t) && res; f++) {
```

Comment:
Me parece que está mal el rango que recorre. Also, el `&& res` es medio raro... Si queremos usarlo deberíamos usar un `while`
```suggestion
        for (int f = 0; f < filas(t) && res; f++) {
```